csharp-basic-topics/TaskCompletedVsReturnInCSharp/TaskCompletedVsReturnInCSharp.Tests/TaskCompletedClassAndReturnClassTests.cs

```diff
@@ -0,0 +1,25 @@
+namespace TaskCompletedVsReturnInCSharpTests
+{
+    public class TaskCompletedClassAndReturnClassTests
+    {
+        [Fact]
+        public void WhenCallingUseTaskCompletedMethodAsync_ThenUseTaskCompleted()
+        {
+            TaskCompletedClass taskCompletedClass = new TaskCompletedClass();
+
+            Task result = taskCompletedClass.UseTaskCompletedMethodAsync();
```

Comment:
This could change to var to be consistent with the other test

Reply:
Okay. Done